README.md

```diff
@@ -43,7 +47,7 @@ Once integrated, your marketing team will be able to better understand your app
 ## Installation
```

Comment:
I think it would make sense to move the set up steps for App Groups into this section similar to how we have the notification service extension instructions.